src/components/dealInDetails/sliderAndTimerComponents/LimitedOffer.tsx

```diff
@@ -38,6 +38,33 @@ const LimitedOffer = () => {
       const { dealSlug } = useParams();
      
 
+
+      const [ days, SetDays] = useState(0);
+      const [hours, setHours] = useState(0)
+      const [minutes, setMinutes] = useState(0);
+      const [seconds, setSeconds] = useState(0);
+    
+      const dates = "2024-03-31T10:59:59.999Z ";
+      const getTime = ()=>{
+        const remainingTime = Date.parse(dates)-Date.now()
```

Comment:
Use `useCallback` Hook, Other wise it will define every time when state change.  If possible use same login in 62 line instead of create seprate functions